pkgs/by-name/va/vapoursynth-znedi3/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  vapoursynth,
+}:
+
+stdenv.mkDerivation {
+  pname = "vapoursynth-znedi3";
+  version = "unstable-2023-07-09";
+
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
+    owner = "sekrit-twc";
+    repo = "znedi3";
+    rev = "68dc130bc37615fd912d1dc1068261f00f54b146";
+    hash = "sha256-QC+hMMfp6XwW4PqsN6sip1Y7ttiYn/xuxq/pUg/trog=";
+  };
+
+  buildInputs = [ vapoursynth ];
+
+  postPatch = ''
+    rm -rf vsxx/vapoursynth
+    ln -s ${vapoursynth}/include/vapoursynth vsxx/vapoursynth
+  '';
+
+  makeFlags =
+    [ "NNEDI3_WEIGHTS_PATH=$(out)/share/nnedi3_weights.bin" ]
```

Apologies. My original suggestion was incorrect. This should however work:

```suggestion
    [ "CPPFLAGS=-DNNEDI3_WEIGHTS_PATH='\"$(out)/share/nnedi3/nnedi3_weights.bin\"'" ]
```